obelisk/cpp/obelisk_cpp/include/obelisk_mujoco_sim_robot.h

```diff
@@ -95,10 +99,11 @@ namespace obelisk {
          */
         rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
         on_activate(const rclcpp_lifecycle::State& prev_state) {
+            RCLCPP_INFO_STREAM(this->get_logger(), "Activating the ObeliskSimRobot robot");
             this->ObeliskSimRobot<ControlMessageT>::on_activate(prev_state);
 
             activation_complete_ = true;
-
+            RCLCPP_INFO_STREAM(this->get_logger(), "Activated the mujoco robot");
```

Comment:
Do we really need both acitvating... and activated? Maybe we can get away with only one of these logging outputs.